src/main/java/uk/gov/hmcts/darts/audio/component/impl/OutboundFileProcessorImpl.java

```diff
@@ -73,8 +73,8 @@ public List<List<AudioFileInfo>> processAudioForDownload(Map<MediaEntity, Path>
 
     @Override
     public List<AudioFileInfo> processAudioForPlaybacks(Map<MediaEntity, Path> mediaEntityToDownloadLocation,
```

Comment:
Please can you change the access for the 2 convert methods added recently to be `private`?
- convertChannelsListToConcatenationsList
- convertChannelsListToFilesList

Reply:
done ( blaming this one on intellij ! )